Sources/Qualification/State/AppQualifyingService.swift

```diff
@@ -61,6 +60,11 @@ final class AppQualifyingService: QualifyingService {
             let appInfo = try await updateService.fetchAppInfo()
             AppEnvironment.updateReleaseFlags(appInfo.releaseFlags)
             
+            guard appInfo.allowAppUsage else {
```

Comment:
It doesn't look like we have testing for these lines ? 👀 
https://sonarcloud.io/component_measures?metric=new_coverage&selected=di-mobile-ios-onelogin-app%3ASources%2FQualification%2FState%2FAppQualifyingService.swift&view=list&pullRequest=210&id=di-mobile-ios-onelogin-app

Reply:
Very true 😅 added now